### DISCUSSION TITLE
DST start and end datetime with luxon

### DISCUSSION THREAD

**Discussion by lacrimal** [moment/luxon#1372] [Q&A]
Hi,

Fo example current DST time and date in my country are 2023-03-27 01:00 GMT and 2023-10-30 02:00 GMT.
Can luxon give DST start and end date and time ?

Thought this is obvious but can't find simple solution for that.

**Comment by diesieben07** [comment]
Unfortunately this information is not exposed by the Intl API and as such Luxon cannot provide it either.
Luxon does not ship its own timezone data.
